jwst/coron/median_replace_img.py

```diff
@@ -68,8 +68,8 @@ def median_replace_img(img_model, box_size, bad_bitvalue):
 
     Parameters
     ----------
-    img_model : DataModel
-        The input image model
+    img_model : CubeModel
+        The input image Cube model
```

Comment:
```suggestion
        The input model
```
image Cube model seems contradictory